src/components/CreateContract/CreateContractForms/team/container.js

```diff
@@ -1,54 +1,37 @@
 import React, { Component } from 'react';
 import { PropTypes as PT } from 'prop-types';
-import { getAllClients } from '../../../../services/clientService';
-import { getTeamsFromClient } from '../../../../services/teamService';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
 
-export default Wrapped =>
+import { updateSelectedTeamId } from '../../../../store/actions/teams';
```

Comment:
using webpack aliases here will tidy up the imports

Reply:
Yes, Mark was showing me this yesterday ands after a bit of research, I totally agree. This should be done in a new ticket.